src/main/java/seedu/algobase/model/problem/Problem.java

```diff
@@ -33,6 +34,7 @@
     public Problem(Name name, Author author, WebLink webLink, Description description, Set<Tag> tags,
                    Difficulty difficulty, Remark remark, Source source) {
         requireAllNonNull(name, author, webLink, description, tags, difficulty, remark, source);
+        this.id = System.currentTimeMillis() / 1000L;
```

Comment:
It's a requirement that every field is human editable and readable - maybe we should ask prof on this implementation?